common/config/rush/pnpm-lock.yaml

```diff
@@ -1,5 +1,5 @@
 dependencies:
-  '@azure/amqp-common': 1.0.0-preview.6
+  '@azure/amqp-common': 1.0.0-preview.6_rhea-promise@0.1.15
```

Comment:
nice

Reply:
Why is this nice?  I don't understand why this changed or what it means.

Reply:
I am assuming this is how it's pinning transitive dependencies? If not, I take back my comment.